types/src/front/lib/core_api.ts

```diff
@@ -77,33 +77,26 @@ type GetDatasetsResponse = {
   datasets: { [key: string]: CoreAPIDatasetVersion[] };
 };
 
-export type CoreAPIDatabase = {
-  created: number;
-  data_source_id: string;
-  database_id: string;
-  name: string;
-};
-
-export type CoreAPIDatabaseTableSchema = Record<
+export type CoreAPITableSchema = Record<
   string,
   "int" | "float" | "text" | "bool" | "datetime"
 >;
 
-export type CoreAPIDatabaseTable = {
+export type CoreAPITable = {
   created: number;
-  database_id: string;
   table_id: string;
+  data_source_id: string;
```

Comment:
Should be dataSourceName?

Reply:
It's an object `returned by `core` so `dataSourceName` wouldn't make sense (that's a `front` thing)